test_runner/performance/test_lfc_prefetch.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
+
+import random
+import threading
+import time
+from typing import TYPE_CHECKING
+
+import pytest
+from fixtures.log_helper import log
+
+if TYPE_CHECKING:
+    from fixtures.neon_fixtures import NeonEnv
+from fixtures.utils import USE_LFC
+
+
+@pytest.mark.remote_cluster
+@pytest.mark.timeout(100000)
+@pytest.mark.parametrize("n_readers", [1, 2, 4, 8])
+@pytest.mark.parametrize("n_writers", [0, 1, 2, 4, 8])
+@pytest.mark.parametrize("chunk_size", [1, 8, 16])
```

Comment:
With the chunk_size parameter added, we have 60 combinations and each runs for 100 seconds, so the test would take 6000 seconds of CPU time. Isn't it too expensive for such a simple (exercising just a couple of operations) test?

Wouldn't it be better to choose the parameters randomly and just print chosen values in the log to ease reproduction?